src/containers/user-profile/comments-with-rating-block/CommentsWithRatingBlock.tsx

```diff
@@ -55,6 +56,28 @@ const CommentsWithRatingBlock = ({
     </MenuItem>
   ))
 
+  const filteredItems = items.filter((item) => {
+    if (filter !== null) {
+      return item.rating === filter
+    }
+    return true
+  })
+
+  const sortedItems = filteredItems.sort((a, b) => {
```

Comment:
Change to ``toSorted`` to make a copy or use filter.sort above

Reply:
done✅